app/src/main/java/org/jellyfin/androidtv/ui/livetv/LiveTvGuideActivity.java

```diff
@@ -260,7 +260,10 @@ protected void onResume() {
     }
 
     protected void doLoad() {
-        if (TvManager.shouldForceReload() || System.currentTimeMillis() > mLastLoad + 3600000) {
+        Calendar curTime = Calendar.getInstance();
+        curTime.setTime(new Date());
```

Comment:
Why did you change this from using `System.currentTimeMillis()`?

Reply:
I changed a lot of these to use Calendar. For some reason, it appears that the calendar.getTimeMs fucntion adds some sort of zero padding and so Calendars current time would not ever equal Systems current time for me. I can look into this further and see if it was just my bad code or a real issue. The mCurrentGuide longs are set based off of Calendar variables.